sw/device/silicon_creator/lib/meson.build

```diff
@@ -311,6 +311,23 @@ mask_rom_tests += {
   }
 }
 
+sw_silicon_creator_lib_sigverify_mod_exp_otbn_functest = declare_dependency(
```

Comment:
@drewmacrae Do you think we should update `sw/device/silicon_creator/lib/BUILD` before merging this PR? It looks like we can already build OTBN binaries in bazel.

Reply:
I'd be happy to see it, I have the otbn smoketest working in this PR: https://github.com/lowRISC/opentitan/pull/9640

Reply:
If it's difficult or problematic we can get this one added to bazel in the future.

Reply:
Looks like it's trying to run the test on CW310 only -- can I create verilator targets as well?

Reply:
I ran it on all non-cw310 targets with the command: `bazel test //... --test_tag_filters=-cw310 --disk_cache=~/bazel_cache --local_test_jobs=HOST_CPUS*0.25` and all the tests we expect passed.